src/protocols/secure_channel/CASESession.cpp

```diff
@@ -1045,36 +982,37 @@ CHIP_ERROR CASESession::HandleSigmaR3(System::PacketBufferHandle & msg)
     err = decryptedDataTlvReader.EnterContainer(containerType);
     SuccessOrExit(err);
 
-    err = decryptedDataTlvReader.FindElementWithTag(CASETLVTag::kNOC, suppTlvReader);
+    err = decryptedDataTlvReader.FindElementWithTag(TLV::ContextTag(1), suppTlvReader);
     SuccessOrExit(err);
     VerifyOrExit(suppTlvReader.GetType() == TLV::kTLVType_ByteString, err = CHIP_ERROR_WRONG_TLV_TYPE);
     responderOpCertLen = static_cast<uint16_t>(suppTlvReader.GetLength());
     err                = suppTlvReader.GetBytes(responderOpCert, responderOpCertLen);
     SuccessOrExit(err);
 
-    // Step 3
+    // Step 5
```

Comment:
This is steps 5/6 together I think:

![image](https://user-images.githubusercontent.com/1313847/126201196-f6222eb8-5761-4efc-bbb7-21374aa543f7.png)

Reply:
Updated.